### ISSUE TITLE
Do I have to specify the port for images with one port exposed?

### ISSUE THREAD

**Issue by slsyy** [testcontainers/testcontainers-go#343]
```dockerfile
# Dockerfile
EXPOSE 11211
CMD ["memcached"]
```

```golang
// Code
port := "11211/tcp"
rq := testcontainers.ContainerRequest{
	Image:        "memcached:1.6.10",
	ExposedPorts: []string{port},
	WaitingFor:   wait.ForListeningPort(nat.Port(port)),
}

memcachedC, err := testcontainers.GenericContainer(context.Background(), testcontainers.GenericContainerRequest{
	ContainerRequest: rq,
	Started:          true,
})

url, _ := memcachedC.Endpoint(context.Background(), "")
```

Is there any way to have the same functionality without specifying the port manually in a code?

**Comment by ajbouh** [comment]
I tried to make this work as well and eventually settled for parsing the container definition itself to discover the port

**Comment by fiftin** [comment]
@slsyy Why not use `NetworkMode: "host` ?

**Comment by mdelapenya** [comment]
I think we can close this one, as https://github.com/testcontainers/testcontainers-go/pull/468 was merged and released in v0.14.0, but please let us know if you need anything else here and feel free to reopen it.

Thanks for your time!
